src/main/java/com/maniacalhunter/DisplayMode.java

```diff
@@ -4,5 +4,6 @@ public enum DisplayMode
 {
 	SESSION_ONLY,
 	AGGREGATE_ONLY,
-	BOTH
+	BOTH,
+	CONDENSED
```

Comment:
condensed should be an option outside of the stats to track, because condensed could still be set to show session data, aggregate data, or both data sets